src/pluraldo/__init__.py

```diff
@@ -51,20 +51,45 @@ async def switch(name):
 
 
 @cli.command()
-@click.argument("project", required=False)
 @entry
-async def workon(project):
+async def clear():
```

Comment:
did you mean `pluraldo project clear` or `pluraldo clear`?

Reply:
I meant `pluraldo clear`, to try to replicate the previous functionality of `pluraldo workon` without an argument clearing the current project.